Centaurus/Alpha/AlphaStartup.cs

```diff
@@ -0,0 +1,165 @@
+            {
+                Configuration = configuration;
+            }
+
+            public IConfiguration Configuration { get; }
+
+            // This method gets called by the runtime. Use this method to add services to the container.
+            // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+            public void ConfigureServices(IServiceCollection services)
+            {
+                services
+                    .AddMvc(options => options.EnableEndpointRouting = false);
+                //.AddJsonOptions(options =>
+                //{
+                //    options.JsonSerializerOptions.Converters.Add(new AssetSettingsConverter());
+                //});
+
+                services.Add(
+                    new ServiceDescriptor(
+                        typeof(IActionResultExecutor<JsonResult>),
+                        Type.GetType("Microsoft.AspNetCore.Mvc.Infrastructure.SystemTextJsonResultExecutor, Microsoft.AspNetCore.Mvc.Core"),
+                        ServiceLifetime.Singleton)
+                );
+            }
+
+            static ApplicationState[] ValidApplicationStates = new ApplicationState[] { ApplicationState.Rising, ApplicationState.Running, ApplicationState.Ready };
+
+            // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+            public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+            {
+                if (env.IsDevelopment())
+                {
+                    app.UseDeveloperExceptionPage();
+                }
+                else
+                {
+                    app.UseExceptionHandler("/Error");
+                    // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+                    app.UseHsts();
+                }
+
+                app.UseStaticFiles();
+
+                app.UseWebSockets();
+
+                app.Use(async (context, next) =>
+                {
+                    if (context.WebSockets.IsWebSocketRequest && context.Request.Path == "/centaurus")
+                    {
+                        if (Global.AppState == null || ValidApplicationStates.Contains(Global.AppState.State))
+                        {
+                            WebSocket webSocket = await context.WebSockets.AcceptWebSocketAsync();
+                            await ConnectionManager.OnNewConnection(webSocket, context.Connection.RemoteIpAddress.ToString());
+                        }
+                        else
+                        {
+                            context.Abort();
+                        }
+                    }
+                    else
+                    {
+                        await next();
+                    }
+                });
+
+                app.UseMvc(routes =>
```

Comment:
It's probably not needed anymore since we don't use MVC pipeline directly and removed all the `cshtml` pages.

Reply:
We do need it. We use it to handle constellation init and constellation info requests.